packages/utilities/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@drupal/admin-ui-utilities",
```

Comment:
What other features are we planning to include in this package?

Reply:
@lauriii There was some discussion in #571 about this. I don't think for now anything outside the api utilities are needed.

Reply:
For me this is ready to be merged.